liquibase/README.md

```diff
@@ -125,7 +125,16 @@ need to be updated so that they include the change.
 The pom file of the openmrs-liquibase module contains a template for generating Liquibase snapshots from an existing 
 database and applying snapshots to an OpenMRS database.
 
-### How to generate Liquibase snapshots
+### How to generate Liquibase snapshots automatically
+#### step 1 - checkout to project root folder
+ 	cd <some root folder>/openmrs-core
+#### step 2 - run the script `openmrs/liquibase/scripts/generate_liquilbase_snapshots.sh`:
+Run the following commands to generate the Liquibase snapshots automatically, where username and password refer to a MySQL user:
+
+	. liquibase/scripts/generate_liquibase_snapshots.sh <username> <password>
```

Comment:
After running this script, can you list the files which will get created and their output locations?